packages/dd-trace/src/appsec/iast/taint-tracking/csi-methods.js

```diff
@@ -2,13 +2,16 @@
 
 const csiMethods = [
   { src: 'concat' },
+  { src: 'join' },
   { src: 'parse' },
   { src: 'plusOperator', operator: true },
   { src: 'random' },
   { src: 'replace' },
   { src: 'slice' },
   { src: 'substr' },
   { src: 'substring' },
+  { src: 'toLowerCase', dst: 'stringCase' },
```

Comment:
I'm curious... Do we also patch things like `String.prototype.toLowerCase`?

Reply:
Yes, we do.